fate_flow/service.sh

```diff
@@ -17,7 +17,7 @@
 #
 
 export PYTHONPATH=
-log_dir="$(echo ${PYTHONPATH} | awk -F":" '{print $1}')/logs"
+log_dir="$(dirname "$PWD")/logs"
```

Comment:
pwd is not reliable, it is recommended to change to get the path of this file

Reply:
done